packages/nextjs/test/run-integration-tests.sh

```diff
@@ -31,10 +31,10 @@ echo "Running integration tests on Node $NODE_VERSION"
 # make a backup of our config file so we can restore it when we're done
 mv next.config.js next.config.js.bak
 
-for NEXTJS_VERSION in 10 11 12 13; do
+for NEXTJS_VERSION in 13; do
   for USE_APPDIR in true false; do
-    if ([ "$NEXTJS_VERSION" -lt "13" ] || [ "$NODE_MAJOR" -lt "16" ]) && [ "$USE_APPDIR" == true ]; then
-      # App dir doesn not work on Next.js < 13 or Node.js < 16
+    if ([ "$NODE_MAJOR" -lt "16" ]) && [ "$USE_APPDIR" == true ]; then
+      # App dir doesn not work on Node.js < 16
```

Comment:
we can remove the `USE_APPDIR` conditional here because Next.js 13 is Node 16+

Reply:
That is not entirely true. Only the app directory requires Node 16. 13.2.0 still partially supports node 14 https://github.com/vercel/next.js/blob/v13.2.0/packages/next/package.json. This is also why we shouldn't update the matrix.